Makefile

```diff
@@ -111,9 +111,9 @@ lint-config: golangci-lint ## Verify golangci-lint linter configuration
 
 .PHONY: yamllint
 yamllint:
-	@files=$$(find testdata -name '*.yaml' ! -path 'testdata/*/dist/*'); \
-    	docker run --rm $$(tty -s && echo "-it" || echo) -v $(PWD):/data cytopia/yamllint:latest $$files -d "{extends: relaxed, rules: {line-length: {max: 120}}}" --no-warnings
-
+	@files=$$(find testdata -name '*.yaml' ! -path 'testdata/*/dist/chart/install.yaml'); \
+	docker run --rm $$(tty -s && echo "-it" || echo) -v $(PWD):/data cytopia/yamllint:latest $$files -d "{extends: relaxed, rules: {line-length: {max: 120}}}" --no-warnings
```

Comment:
@sarthaksarthak9 

Please, squash the commits. 
In order to keep a better git history and allow us to revert things if we need to do so
We should have ONLY 1 commit for this change

Can you please do that to allow us to move forward here?